test/spec/spectest.cpp

```diff
@@ -46,34 +36,37 @@ using namespace WasmEdge;
 
 // Preprocessing for set up aliasing.
 void resolveRegister(std::map<std::string, std::string> &Alias,
-                     rapidjson::Value &CmdArray,
-                     rapidjson::Document::AllocatorType &Allocator) {
-  rapidjson::Value::ValueIterator ItMod = CmdArray.Begin();
-  for (rapidjson::Value::ValueIterator It = CmdArray.Begin();
-       It != CmdArray.End(); ++It) {
-    const auto CmdType = It->GetObject()["type"].Get<std::string>();
+                     simdjson::dom::array CmdArray) {
+  std::string_view OrgName;
+  uint64_t LastModLine;
+  for (simdjson::dom::object Cmd : CmdArray) {
+    std::string_view CmdType = Cmd["type"];
+    bool Replaced = false;
     if (CmdType == "module"sv) {
-      // Record last module in order.
-      ItMod = It;
+      // Record last module in order
+      if (Cmd["name"].get(OrgName)) {
+        OrgName = {};
+      }
+      LastModLine = Cmd["line"];
     } else if (CmdType == "register"sv) {
-      const auto NewNameStr = It->GetObject()["as"].Get<std::string>();
-      auto OrgName = ItMod->FindMember("name");
```

Comment:
The original code retrieves the `OrgName`. However, your new code doesn't do the same thing.

Reply:
Explained below.